src/layers/MultiscaleImageLayer/MultiscaleImageLayer.js

```diff
@@ -172,8 +169,7 @@ export default class MultiscaleImageLayer extends CompositeLayer {
         z: numLevels - 1,
         pickable: true,
         onHover,
-        onClick,
-        boxSize: getNearestPowerOf2(lowResWidth, lowResHeight)
```

Comment:
Can remove import for `getNearestPowerOf2` as well?

Reply:
Weird that linting didn't catch that....thanks.